packages/agoric-cli/src/start.js

```diff
@@ -630,7 +637,7 @@ export default async function startMain(progname, rawArgs, powers, opts) {
               SOLO_COINS,
             ],
           ];
-          for (const cmd of provCmds) {
+          for (/* await */ const cmd of provCmds) {
```

Comment:
The await is commented out because
    * turning it into a for-await-of loop would legitimize some `await` in its body
    * both a for-await-of loop is not allowed here. This needs repair first.